ISSUE_TEMPLATE

```diff
@@ -0,0 +1,17 @@
+### Context
+
+* [ ] **mapbox-gl-js version**: 
+* [ ] **browser version**: 
+* [ ] **operating system**: 
+
+### Report
+
+**Expected behavior**:
+
+**Actual behavior**:
+
+**Steps to reproduce the behavior**:
+
+1.
+2.
+3.
```

Comment:
These are great suggestions for the `CONTRIBUTING.md` topic, but they don't make sense within the context of an issue template.